primer3/thermoanalysis.pyx

```diff
@@ -25,14 +25,14 @@ calculations using common calculation parameters.
 
 Calculations are performed under the following paradigm:
 
-1) Instantiate ThermoAnalysis object with appropriate parameters
+1) Instantiate _ThermoAnalysis object with appropriate parameters
```

Comment:
Looks like a global find-and-replace operation hit some of the documentation (I would assume that this and below should reference `ThermoAnalysis` and not `_ThermoAnalysis`

Reply:
fixed a couple of these